src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -13,15 +13,17 @@
 
 import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.HashSet;
 import java.util.List;
 
 import seedu.address.commons.core.index.Index;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.AddressBook;
 import seedu.address.model.Model;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
-import seedu.address.model.person.Person;
+import seedu.address.model.person.*;
```

Comment:
As mentioned prior, perhaps try to avoid the use of * imports, noticed the use elsewhere with the import of CommandTestUtil.* as well, under RemoveCommandTest.java